pkg/column-mapping/column.go

```diff
@@ -490,22 +495,33 @@ func computePartitionID(schema, table string, rule *Rule) (instanceID int64, sch
 		instanceID = int64(instanceIDUnsign << shiftCnt)
 	}
 
+	var sep string
+	if len(rule.Arguments) > 3 {
```

Comment:
can we only do this check _once_ for one rule? like do it in `AddRule`/`UpdateRule `, not in `computePartitionID`.